src/webui/www/private/scripts/prop-trackers.js

```diff
@@ -155,9 +106,17 @@ var torrentTrackersContextMenu = new ContextMenu({
         y: 2
     },
     onShow: function() {
-        var element = this.options.element;
-        selectedTracker = element;
-        if (element.childNodes[1].innerText.indexOf("** [") === 0) {
+        var selectedTrackers = torrentTrackersTable.selectedRowsIds();
+
+        var containsStaticTracker = false;
```

Comment:
If `selectedTrackers` is an array, then it should be possible to write:
```js
var containsStaticTracker = selectedTrackers.some(function (tracker) {
    return (tracker.indexOf("** [") === 0);
});
```

Reply:
I didn't know about `some`. This is much cleaner.